suite-native/assets/src/components/Assets.tsx

```diff
@@ -39,6 +39,29 @@ export const Assets = () => {
         selectAssetsWithBalances(state, fiatCurrency.label),
     );
 
+    const assetsDataWithPercentage = useMemo(() => {
```

Comment:
I would either create new selector that inherits data from `selectAssetsWithBalances` or extend existing selector to add percentage as an asset data property.

Reply:
This is purely UI thing related to order in which items are displayed and this will be complicated with selector if you will start filtering of changing order (and this will probably happen because we should use same icon basically everywhere incl. accounts list)

Reply:
I think maybe moving `fiatPercentage` to selector could make sense, since it's not order related, but `fiatPercentageOffset` should be calculated on UI side.

Reply:
Yeah, I meant adding the fiatPercentage to the properties of asset data object. I see no reason why not include it. All of the stuff in assets selectors is UI related by definition because it's just accounts grouped together to make it easier to understand for users in the UI. 

```
 const asset: AssetType = {
                    symbol: networkSymbol,
                    network,
                    assetBalance: formattedAssets[networkSymbol] ?? new BigNumber(0),
                    fiatBalance,
                    fiatPercentage: // Add function for calculating percentage
                };
```

Every time we will be getting asset data we will also need to know the percentage because it's UI related as you mentioned so I would say it makes sense to add it here.
cc @Nodonisko

Reply:
Makes sense, but current solution is good enough. I will create follow-up task for using this also in account list, so it could be refactored then if it makes sense.